bootstrap/config/config.go

```diff
@@ -180,6 +181,13 @@ func (cp *Processor) Process(
 
 	// Now must load configuration from local file if any of these conditions are true
 	if !useProvider || !cp.providerHasConfig || cp.overwriteConfig {
+		commonConfigLocation := environment.GetCommonConfig(cp.lc, cp.flags.CommonConfig())
+		if commonConfigLocation != "" {
+			err := cp.loadCommonConfigFromUri(commonConfigLocation, serviceConfig, serviceType)
```

Comment:
Lets not use UIR term yet. When we do implement URI if will be a generic function not specific to configuration. My thought is this loadCommonConfigFromFile will can it to do the actually loading once we have it in place.
```suggestion
			err := cp.loadCommonConfigFromFile(commonConfigLocation, serviceConfig, serviceType)
```